django-server/README.md

```diff
@@ -54,4 +54,16 @@ python manage.py dumpdata --exclude=auth --exclude=contenttypes --exclude=sessio
 and then restored in the other environment using the 
 ```bash
 python manage.py loaddata data.json
-```
\ No newline at end of file
+```
+
+## Sample data
+
+To load sample data in empty project, run:
+```
+python manage.py migrate
+python manage.py loaddata fixtures/demo-data.yaml
+python manage.py reset_passwords  # Set password equal to username for each user
```

Comment:
not sure why do we need this and if it should be available for such system

Reply:
Here I added it just so you can add user by hand and don't have to run all hashing etc. It's not a problem in terms of security: if you have shell access, then you can do the same thing by hand using `python manage.py shell`.

Generally, Django passwords are salted. If salt uses configured security key, then you can't guarantee your local passwords will be the same as in prod: https://docs.djangoproject.com/en/3.0/topics/auth/passwords/#how-django-stores-passwords